src/service-worker.ts

```diff
@@ -36,7 +36,7 @@ const fileExtensionRegexp = new RegExp('/[^/?]+\\.[^/]+$');
 registerRoute(({ request, url }: { request: Request; url: URL }) => {
   // If this isn't app.uniswap.org, skip. IPFS gateways may not have domain
   // separation, so they cannot use App Shell-style routing.
-  if (url.hostname !== 'app.uniswap.org') {
+  if (url.hostname !== 'quickswap.exchange') {
```

Comment:
You probably need to include the alpha or dev here.

Reply:
made update to check if the hostname is not alpha.quickswap.exchange or dev.quickswap.exchange

Reply:
I don't see that change @totop716 ?

Reply:
Just pushed a commit with the update